File: Algorithm/subjects/1601-1700/1608. 特殊数组的特征值/main.java
class Solution {
    public int specialArray(int[] nums) {
        for (int i = 1; i <= nums.length; i++) {
            int ans = 0;
            for (int num : nums) {
                if (num >= i) {
                    ans++;
                }
            }
            if (ans == i) {
                return i;
            }
        }
        return -1;
    }
}

File: Algorithm/subjects/1601-1700/1608. 特殊数组的特征值/main.cpp
class Solution {
public:
    int specialArray(vector<int>& nums) {
        int n = nums.size();
        int target = 0;
        for (int i = 1; i <= n; i++) {
            int temp = 0;
            for (int j = 0; j < n; j++) {
                if (nums[j] >= i) {
                    temp++;
                }
            }
            if (temp == i) {
                return temp;
            } 
        }
        return -1;
    }
};